test/com/google/javascript/jscomp/CollapsePropertiesTest.java

```diff
@@ -61,6 +61,10 @@ public void testCollapse() {
   public void testMultiLevelCollapse() {
     test("var a = {}; a.b = {}; a.b.c = {}; var d = a.b.c;",
          "var a$b$c = {}; var d = a$b$c;");
+
+    test("var a = {}; a.b = {}; " +
+        "/** @nocollapse */ a.b.c = {}; var d = a.b.c;",
```

Comment:
The internal linter is complaining here that the + should be on the beginning of the next line, instead of the end of the previous line.